src/tests/accommodation/createAccommodationFacility.test.js

```diff
@@ -194,6 +210,27 @@ export const editAccommodations = () => {
   });
 };
 
+export const SuppliersEditAccommodations = () => {
+  describe('supplier can edit their accomodation ', () => {
+    before((done) => {
+      localStorage.setItem('token', supplierToken);
+      done();
+    });
+
+    it('it should return 200 when editing single accommodation ', (done) => {
+      chai
+        .request(app)
+        .patch(`/api/v1/edit/accommodation/${2}`)
+        .send(editAccommodation)
+        .end((err, res) => {
+          expect(res.statusCode).to.equal(200);
+          expect(res.body.data).to.be.an('object');
```

Comment:
@Kagorora , you did a wonderful job here.
As a suggestion, I think you might consider expecting some of the object's variables/values to make the test more firm.

Reply:
@T2Wil  as you can see below this comment on line 230 and 231 I have implemented your feedback